### ISSUE TITLE
imdb.load_data() is not returning meaningful sentences as I have given print like x_train[0], x_train[1] after loading the data! 

### ISSUE THREAD

**Issue by tawsifsazid** [tensorflow/tensorflow#42884]
<em>Please make sure that this is a bug. As per our
[GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md),
we only address code/doc bugs, performance issues, feature requests and
build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary):
- TensorFlow version (use command below):
- Python version:
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:
- GPU model and memory:

You can collect some of this information using our environment capture
[script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh)
You can also obtain the TensorFlow version with:
1. TF 1.0: `python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"`
2. TF 2.0: `python -c "import tensorflow as tf; print(tf.version.GIT_VERSION, tf.version.VERSION)"`

**Describe the current behavior**
keras imdb.load_data() method is not returning meaningful sentence as a row after loading in x_train, y_train, x_test, y_test.

**Describe the expected behavior**
It should have an actual meaningful review sentence from the dataset.

**Standalone code to reproduce the issue**
Provide a reproducible test case that is the bare minimum necessary to generate
the problem. If possible, please share a link to Colab/Jupyter/any notebook.
`from keras.datasets import imdb
row = ''
(x_train, y_train), (x_test, y_test) = imdb.load_data()
word_to_ix = imdb.get_word_index()
#print(word_to_ix['hello'])
ix_to_word = dict( (value, key) for key, value in word_to_ix.items() )
#print(ix_to_word[4822])

for i in range(len(x_train[0])):
  row += ix_to_word[x_train[0][i]] + ' '
print(row)`
[https://colab.research.google.com/gist/tawsifsazid/ef2b113e487c233f5ebd8cbddc95ef75/bug_maybe.ipynb?authuser=1](url)
**Other info / logs** Include any logs or source code that would be helpful to
diagnose the problem. If including tracebacks, please include the full
traceback. Large logs and files should be attached.

**Comment by tawsifsazid** [comment]
https://colab.research.google.com/gist/tawsifsazid/ef2b113e487c233f5ebd8cbddc95ef75/bug_maybe.ipynb?authuser=1

**Comment by nikitamaia** [comment]
Hi @tawsifsazid, the indicies are offset by three to account for "padding", "start of sequence" and "unknown". Please take a look at [this StackOverflow thread](https://stackoverflow.com/questions/42821330/restore-original-text-from-keras-s-imdb-dataset) for more information. You should be able to use the following line of code to reconstruct the sentences `' '.join([ix_to_word.get(i - 3, 'UNK') for i in x_train[0]])` where 'UNK' indicates an out of vocabulary word.

Closing this issue now since there is no bug.
